engine/installation/index.md

```diff
@@ -53,6 +53,22 @@ platform.
 See also [Docker Cloud](#on-docker-cloud) for setup instructions for
 Digital Ocean, Packet, SoftLink, or Bring Your Own Cloud.
 
+## Time-based release schedule
+
+Starting with Docker 17.03, Docker uses semantic versioning and a time-based
+release schedule, outlined below. There may be times when exceptions need to be
+made, but this is the goal.
+
+- Ideally, a Docker CE release is published the first week of each month.
```

Comment:
I don't think we should raise expectations that it's published the first week; yes, it's our goal, but not a "contract". We have a "monthly" release, that's the only thing we need to mention